.npmrc

```diff
@@ -4,4 +4,4 @@
 save-workspace-protocol=false
 prefer-workspace-packages=true
 
-engine-strict=true
+engine-strict=false
```

Comment:
scalar got engine.node = 20 🤷🏼‍♂️ 

maybe we should update this repo to also use 20? it's the LTS afterall

Reply:
Now that it is supported on Vercel, sgtm 👍

Reply:
#557